src/Module/Contact/Poke.php

```diff
@@ -0,0 +1,164 @@
+<?php
+
+namespace Friendica\Module\Contact;
+
+use Friendica\BaseModule;
+use Friendica\Core\Hook;
+use Friendica\Core\Logger;
+use Friendica\Core\Renderer;
+use Friendica\Core\System;
+use Friendica\Database\DBA;
+use Friendica\DI;
+use Friendica\Model;
+use Friendica\Network\HTTPException;
+use Friendica\Protocol\Activity;
+use Friendica\Util\XML;
+
+class Poke extends BaseModule
+{
+	public static function post(array $parameters = [])
+	{
+		if (!local_user() || empty($parameters['id'])) {
+			return self::postReturn(false);
+		}
+
+		$uid = local_user();
+
+		if (empty($_POST['verb'])) {
+			return self::postReturn(false);
+		}
+
+		$verb = $_POST['verb'];
+
+		$verbs = DI::l10n()->getPokeVerbs();
+		if (!array_key_exists($verb, $verbs)) {
+			return self::postReturn(false);
+		}
+
+		$activity = Activity::POKE . '#' . urlencode($verbs[$verb][0]);
+
+		$contact_id = intval($parameters['id']);
+		if (!$contact_id) {
+			return self::postReturn(false);
+		}
+
+		Logger::info('poke: verb ' . $verb . ' contact ' . $contact_id);
```

Comment:
Could you remove the "poke:" from the logging string? We did this long before to see the current module.

Reply:
Absolutely.

Reply:
Done.